Moovies/Moovies/DescriptionModule/MoovieDescriptionTableViewController.swift

```diff
@@ -0,0 +1,88 @@
+// MoovieDescriptionTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+class MoovieDescriptionTableViewController: UITableViewController {
+    // MARK: Enum
+
+    enum Cells {
+        case poster
+        case overview
+    }
+
+    // MARK: Internal Properties
+
+    var movieID = Int()
```

Comment:
```suggestion
    var movieID = Int()
```
в презентере храним это